internal/drand-cli/dkg_cli.go

```diff
@@ -172,17 +171,13 @@ var genesisTimeFlag = &cli.StringFlag{
 	Usage: "The duration from now until the network should start creating randomness",
 }
 
-var transitionTimeFlag = &cli.StringFlag{
-	Name:  "transition-delay",
-	Usage: "The duration from now until which keys generated during the next DKG should be used. It will be modified to the nearest round.",
-}
-
 var dkgTimeoutFlag = &cli.StringFlag{
 	Name:  "timeout",
 	Usage: "The duration from now in which DKG participants should abort the DKG if it has not completed.",
 	Value: "24h",
 }
 
+//nolint:dupl//no it's not??
```

Comment:
the first 7 lines are I guess

Reply:
true, though there's not a nice way to extract them - the linter actually showed more than just the 7 lines (though maybe I missed some highlighting?)